exercises/src/test/scala/GettingStartedSpec.scala

```diff
@@ -0,0 +1,77 @@
+import org.specs2.mutable.Specification
+import fpinscala.gettingstarted._
+
+class MyModuleSpec extends Specification {
+  "fib" >> {
+    "0th" >> {
+      MyModule.fib(0) mustEqual 0
+    }
+
+    "1st" >> {
+      MyModule.fib(1) mustEqual 1
+    }
+
+    "nth" >> {
+      // TODO: convert to ScalaCheck style at some point.
+      val n = 5

```

Comment:
@yawaramin short of scalacheck i would do exhaustive checking till some n, say n = 5. you could just loop through them (i.e. the smallcheck/alloy approach)